tests/acceptance/features/webUIFavorites/favoritesFile.feature

```diff
@@ -73,3 +73,20 @@ Feature: Mark file as favorite
     And folder with path "simple-empty-folder" should be listed in the favorites page on the webUI
     And folder with path "simple-folder/simple-empty-folder" should be listed in the favorites page on the webUI
     And file with path "strängé नेपाली folder/lorem.txt" should be listed in the favorites page on the webUI
+
+  @issue-1720
+  Scenario: Try to favorite file and folder that does not exist
+    Given the user has browsed to the files page
+    And the following files have been deleted by user "user1"
+      | name          |
+      | lorem.txt     |
+      | simple-folder |
+    When the user marks file "lorem.txt" as favorite using the webUI
+    #    Then the error message 'Error while marking "lorem.txt" as favorite' should be displayed on the webUI
+    Then file "lorem.txt" should not be marked as favorite on the webUI
+    When the user clears all error message from the webUI
```

Comment:
Is error message even displayed here?

Reply:
Not currently but when the issue is fixed it should be displayed.